app.py

```diff
@@ -107,6 +107,42 @@ def add_ekubo_liquidity(
     return data
 
 
+def create_stablecoin_bundle(data: pandas.DataFrame):
+    for collateral in src.settings.COLLATERAL_TOKENS:
+        relevant_pairs = [pair for pair in data.keys() if collateral in pair and any(stablecoin in pair for stablecoin in src.settings.DEBT_TOKENS[:-1])]
+        combined_df = None
+
+        for pair in relevant_pairs:
+            df = data[pair]
+
+            if 'collateral_token_price' not in df.columns:
```

Comment:
collateral_token_price column cant be missing from preprocessed data. For certain pairs there is no liquidable debt data as platform does not list one or both token in this pair. For instance, Nostra Alpha does not allow borrowing of DAI V2. 
In this case `nostra_alpha_main_chart_data['ETH-DAI V2']` is empty DataFrame.
Therefore more suitable check here would be 
```python
if df.empty:
```
Warning has to be adjust appropriately.

Reply:
Done! ✅